tools/cell_census_builder/manifest.py

```diff
@@ -65,15 +65,15 @@ def load_manifest_from_CxG() -> List[Dataset]:
         }
         for collection in collections
         for dataset in collection["datasets"]
-        if dataset["tombstone"] is False  # ignore anything that has been deleted
     }
     logging.info(f"Found {len(datasets)} datasets, in {len(collections)} collections")
 
     # load per-dataset schema version
-    with concurrent.futures.ThreadPoolExecutor(max_workers=8) as tp:
+    with concurrent.futures.ThreadPoolExecutor(max_workers=4) as tp:
         dataset_metadata = tp.map(
             lambda d: fetch_json(
-                f"{CXG_BASE_URI}curation/v1/collections/{d['collection_id']}/datasets/{d['dataset_id']}"
+                f"{CXG_BASE_URI}curation/v1/collections/{d['collection_id']}/datasets/{d['dataset_id']}",
+                delay_secs=1
```

Comment:
not actually sure if the max_workers or delay was the key to success, but even together this adds insignificant time to the overall build, so not tuning further for now.  Data Portal team has been notified: https://github.com/chanzuckerberg/single-cell-data-portal/issues/3535#issuecomment-1401114636

Reply:
Editorializing: that bug in the underlying service really needs to be fixed!

Reply:
referenced issue in source code